cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -435,16 +436,29 @@ fn send_weth_asset_from_asset_hub_to_ethereum() {
 				RuntimeEvent::ForeignAssets(pallet_assets::Event::Issued { .. }) => {},
 			]
 		);
-		let assets = vec![Asset {
-			id: AssetId(Location::new(
-				2,
-				[
-					GlobalConsensus(Ethereum { chain_id: CHAIN_ID }),
-					AccountKey20 { network: None, key: WETH },
-				],
-			)),
-			fun: Fungible(WETH_AMOUNT),
-		}];
+
+		let assets = vec![
+			Asset {
+				id: AssetId(Location::new(
+					2,
+					[
+						GlobalConsensus(Ethereum { chain_id: CHAIN_ID }),
+						AccountKey20 { network: None, key: WETH },
+					],
+				)),
+				fun: Fungible(WETH_FEE_AMOUNT),
+			},
+			Asset {
+				id: AssetId(Location::new(
+					2,
+					[
+						GlobalConsensus(Ethereum { chain_id: CHAIN_ID }),
+						AccountKey20 { network: None, key: WETH },
+					],
+				)),
```

Comment:
Two kinds of assets with the first as fee in WETH and the second the transferred token.